widgets/deployments/src/ExecutionProgress.tsx

```diff
@@ -0,0 +1,32 @@
+interface ExecutionProgressProps {
+    execution: {
+        status: string;
+        /* eslint-disable camelcase */
+        finished_operations: number;
+        total_operations: number;
+        /* eslint-enable camelcase */
+    };
+}
+
+const ExecutionProgress: React.FunctionComponent<ExecutionProgressProps> = ({ execution }) => {
+    const { Progress } = Stage.Basic;
+    const { Execution } = Stage.Utils;
+
+    const color = Execution.isActiveExecution(execution) ? 'yellow' : 'green';
```

Comment:
Optional:

To show that `yellow` and `green` are not arbitrary strings, we could do:

```suggestion
    const color: SemanticCOLORS = Execution.isActiveExecution(execution) ? 'yellow' : 'green';
```

which would require an `import type` from `semantic-ui-react'` :slightly_smiling_face:

Reply:
Sure, make sense, added 👍🏻